src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -54,5 +54,7 @@
             <argument /> <!-- default lifetime -->
         </service>
 
+        <service id="cache.adapter.null" class="Symfony\Component\Cache\Adapter\NullAdapter" />
```

Comment:
give that we provide no `NullLogger` and the same argument would apply, I still think we shouldn't provide this service :)